internal/provider/vsphere/consts.go

```diff
@@ -0,0 +1,23 @@
+package vsphere
+
+const (
+	PhCluster          = "clusterplaceholder"
+	PhVcenter          = "vcenterplaceholder"
+	PhNetwork          = "networkplaceholder"
+	PhDefaultDatastore = "defaultdatastoreplaceholder"
+	PhUsername         = "usernameplaceholder"
+	PhPassword         = "passwordplaceholder"
+	PhDatacenter       = "datacenterplaceholder"
+
+	DbFieldVsphere          = "platform_vsphere_username"
```

Comment:
This field is wrong and probably not needed

Reply:
This is part of the initial vsphere implementation, I didn't want to touch any of the original logic (at least to try and change as little as I can).
If you feel this should be removed then please create a followup pr that removes this field and it can be discussed there